server/src/index.js

```diff
@@ -0,0 +1,52 @@
+require('dotenv').config();
+const express = require('express');
+const app = express();
+const morgan = require('morgan');
+const bodyParser = require('body-parser');
+const path = require('path');
+const cors = require('cors');
+const mongoose = require('mongoose');
+const expressJwt = require('express-jwt');
+
+const server = require('http').Server(app);
+
+// MongoDB connection configuration
+mongoose.Promise = global.Promise;
+mongoose.connect(process.env.DB_URL, {useMongoClient: true}, (err, res) => {
+  if (err)
+    console.log(`err connecting to db on ${process.env.DB_URL}, err: ${err}`);
+  else
+    console.log(`----- Database connected on ${process.env.DB_URL} -----`);
+}); // connect to our database
```

Comment:
one more think regarding Promise. Usually it's necessary to sync our db connection with our server that's why we need a promise above. so common practice is
```
mongoose.connect(cb)
  .then(startServer)
  .catch(somethingWrong)
```